cmd/launcher/options.go

```diff
@@ -178,22 +178,40 @@ func parseOptions(args []string) (*launcher.Options, error) {
 		return nil, err
 	}
 
+	// Set control server URL and control server TLS settings based on Kolide server URL, defaulting to local server
+	controlServerURL := "localhost:3000"
+	insecureTLS := *flInsecureTLS
+	disableControlTLS := false
+	if *flKolideServerURL == "k2device.kolide.com" {
+		controlServerURL = "k2control.kolide.com"
+	} else if *flKolideServerURL == "k2device-preprod.kolide.com" {
+		controlServerURL = "k2control-preprod.kolide.com"
+	} else if strings.HasSuffix(*flKolideServerURL, "herokuapp.com") {
+		controlServerURL = *flKolideServerURL
+	} else if *flKolideServerURL == "localhost:3443" {
+		controlServerURL = *flKolideServerURL
+		insecureTLS = true
+	} else if *flKolideServerURL == "localhost:3000" {
+		controlServerURL = *flKolideServerURL
+		disableControlTLS = true
+	}
```

Comment:
If we don't match any of these (a user running launcher outside kolide) we should disable the control server.

Reply:
`Control` is already hardcoded to false below (and the corresponding flag is marked as deprecated above) -- but I don't think that flag is actually in use right now. Is there another way you meant for disabling the control server?

Reply:
Hrm. I think there's some out of date bits here. 

The intent, is to run the control server, with those hostnames. If launcher is not talking to one of these, no control server.

I think opts.Control isn't used any more. So in the default case, launcher would try to hit localhost. Which doesn't seem like what we want.

I think either we plumb opts.Control back. or we remove it, and do something with `controlServerURL` being `""`

Reply:
@seejdev do you have a suggestion? Otherwise I'm inclined to make `opts.Control` do something.

I think currently it will try to hit localhost by default, so I can also leave it as-is (no behavior change) for this PR and change it up in a subsequent PR once we know how we'd like this to work.

Reply:
I think my bias would be to remove `opts.Control`, and just check for whether `controlServerURL` is empty. No real need for 2 variables.

Reply:
Could probably kick that to the next PR. Once `krypto` merges, we'll want to update the calling functions

Reply:
Discussed offline + updated accordingly -- now we don't run the control service if `controlServerURL` is empty, and also updated `krypto`